.env

```diff
@@ -1,2 +1,10 @@
 RACK_ENV=development
 REDISTOGO_URL=redis://localhost:6379/
+
+# Configure for member center
+MC_HOST=mcenter.internal.worksap.com
+MC_APP_ID=rhinobird
+
+# TODO-remove:  remove those key-value pair and set them in environment variables
+MC_SECRET_KEY=Djzf8Wh4QBxZjXMh
+MC_CREDENTIAL=6xgauDZP61vgs7AqJMj6QbKKzFxXKkTo
```

Comment:
Never commit keys into source code, you have to remove it through the whole commit tree and replace every commit related.